frontend/src/components/common/button/button.tsx

```diff
@@ -1,36 +1,51 @@
 import { AppRoute } from 'common/enums/enums';
-import { FC } from 'common/types/types';
+import { FC, IconName } from 'common/types/types';
 import { getValidClasses } from 'helpers/helpers';
+import { ReactElement } from 'react';
 
-import { Link } from '../common';
+import { Icon, Link } from '../common';
 import styles from './styles.module.scss';
 
 type Props = {
-  label: string;
-  btnColor?: 'blue' | 'gray';
+  label?: string;
+  btnColor?: 'blue' | 'gray' | 'transparent';
```

Comment:
```suggestion
  btnColor?: 'blue' | 'gray';
```
you don't need it since we should be able to paint the icon